actionview/.gitignore

```diff
@@ -1,4 +1,5 @@
 /lib/assets/compiled/
+/test/ujs/compiled/
```

Comment:
Is there a reason we're compiling to a new location? Would it not be possible to use the existing `app/assets/javascripts/rails-ujs.js`?

Reply:
This is compiling the test file which we do not want to have in `app/assets/javascripts/` as it is not meant to be part of the javascript that we ship with the gem.

This was based on the implementation of the tests in Action Cable. I am not sure if the compilation of the test scripts is strictly required - and there may be a way of running them without this build step - but coping Action Cable seemed like the easiest path to getting to something that is good enough.

Reply:
> This is compiling the test file which we do not want to have in `app/assets/javascripts/` as it is not meant to be part of the javascript that we ship with the gem.

Ack, now I'm seeing that it's compiling something different. Also agree that Action Cable is probably a good resource to look at